src/bin/merchant/close.rs

```diff
@@ -25,14 +27,17 @@ impl Close {
     pub async fn run(
         &self,
         config: &Config,
+        service: &Service,
         merchant_config: &MerchantConfig,
         chan: Chan<protocol::Close>,
     ) -> Result<(), anyhow::Error> {
         let database = database(config).await?;
 
         // Run zkAbacus close and update channel status to PendingClose
         let (chan, close_state) = zkabacus_close(merchant_config, database.as_ref(), chan)
+            .with_timeout(3 * service.message_timeout)
```

Comment:
Why is this 3 times the message_timeout?

Reply:
The mutual close protocol has three messages sent: a signature on the close state, the close state itself, and an abort indicator from the merchant after they check correctness of those two items. The `message_timeout` is supposed to be the maximum amount of time we expect a single message to take to get from one party to the other. The assumption I made is that the internal computation time (e.g. to validate the close state signature) will be negligible compared to the network time.